### PR TITLE
Add documentation for saved object management to support multiple datasource

### PR DESCRIPTION
**Pull request by BionIT**
### Description
When multiple data source feature is turned on, user can upload file which stores saved objects from a remote datasource and the dashboard can recognize and use the saved objects. We are releasing this feature in Open Search dashboards 2.12. This change adds documentation about how to use this feature under multiple data source section.

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/6345

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _dashboards/management/multi-data-sources.md [hunk 1]
```diff
@@ -89,9 +89,23 @@ To select a data source through the Dev Tools console, follow these steps:
...
 
+### Import saved objects from connected data sources into dashboard via file upload within Saved Object Management
+To get saved objects from connected datasources into the current dashboard, you can export the saved objects from connected datasource into ndjson file via **Saved object management** page, then go to **Saved object management** page of the current dashboard which has multiple datasource feature enabled to upload the file. This option provides smooth migration of saved objects from dashboards you currently have to a new dashboard which has multiple datasource enabled. 
+
+Watch the following short video to see it in action.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/import_saved_objects_with_file_upload.gif" alt="Multiple data sources in Saved Object Management">{: .img-fluid}
+
+To import saved objects belong to a connected data source, follow these steps:
+1. Locate your copy of `opensearch_dashboards.yml` and open it in the editor of your choice.
```
Comment (vagimeli) [inline _dashboards/management/multi-data-sources.md:103]:
```suggestion
1. Locate your `opensearch_dashboards.yml` file and open it in your preferred text editor. 
```

File: _dashboards/management/multi-data-sources.md [hunk 2]
```diff
@@ -89,9 +89,23 @@ To select a data source through the Dev Tools console, follow these steps:
...
+To get saved objects from connected datasources into the current dashboard, you can export the saved objects from connected datasource into ndjson file via **Saved object management** page, then go to **Saved object management** page of the current dashboard which has multiple datasource feature enabled to upload the file. This option provides smooth migration of saved objects from dashboards you currently have to a new dashboard which has multiple datasource enabled. 
+
+Watch the following short video to see it in action.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/import_saved_objects_with_file_upload.gif" alt="Multiple data sources in Saved Object Management">{: .img-fluid}
+
+To import saved objects belong to a connected data source, follow these steps:
+1. Locate your copy of `opensearch_dashboards.yml` and open it in the editor of your choice. 
+2. Set `data_source.enabled` to `true`.
+3. Connect to OpenSearch Dashboards and select **Dashboards Management** > **Saved objects**.
```
Comment (vagimeli) [inline _dashboards/management/multi-data-sources.md:105]:
```suggestion
3. Connect to OpenSearch Dashboards and go to **Dashboards Management** > **Saved objects**.
```

File: _dashboards/management/multi-data-sources.md [hunk 3]
```diff
@@ -89,9 +89,23 @@ To select a data source through the Dev Tools console, follow these steps:
...
+4. Click **Import** link and in **Select file**, upload the file acquired from a connected data source.
```
Comment (vagimeli) [inline _dashboards/management/multi-data-sources.md:106]:
```suggestion
4. Select **Import** > **Select file** and upload the file acquired from the connected data source.
```

File: _dashboards/management/multi-data-sources.md [hunk 4]
```diff
@@ -89,9 +89,23 @@ To select a data source through the Dev Tools console, follow these steps:
...
+5. From the **Data source** dropdown menu, select the data source where the file is acqured and then choose the **Conflict management** option and then click **Import** button.
```
Comment (vagimeli) [inline _dashboards/management/multi-data-sources.md:107]:
```suggestion
5. Choose the appropriate **Data source** from the dropdown menu, set your **Conflict management** option, and select the **Import** button.
```

**Comment by vagimeli** [comment]
@BionIT Thank you for putting up the PR. Please see my doc review changes. I'll schedule this for editorial review.

**Comment by BionIT** [comment]
> @BionIT Thank you for putting up the PR. Please see my doc review changes. I'll schedule this for editorial review.

Thanks @vagimeli ! Should I commit your suggestions at the same time?

**Comment by natebower** [comment]
@vagimeli Please commit suggestions and address Vale comments and tag me when ready for editorial. Thanks!

**Comment by vagimeli** [comment]
@natebower Thanks! I addressed Vale issues in my rewrites. I'll sync with tech reviewer to accept my edits and then follow up with you.

**Comment by vagimeli** [comment]
> > @BionIT Thank you for putting up the PR. Please see my doc review changes. I'll schedule this for editorial review.
> 
> Thanks @vagimeli ! Should I commit your suggestions at the same time?

@BionIT Yes, if you are good with my rewrites/edits, you can select the Sign off and commit suggestion button. This lets me know you've approved the text changes. Thank you :)

**Comment by BionIT** [comment]
> > > @BionIT Thank you for putting up the PR. Please see my doc review changes. I'll schedule this for editorial review.
> > 
> > 
> > Thanks @vagimeli ! Should I commit your suggestions at the same time?
> 
> @BionIT Yes, if you are good with my rewrites/edits, you can select the Sign off and commit suggestion button. This lets me know you've approved the text changes. Thank you :)

Thanks @vagimeli ! signed off for all your suggestions!

**Review by vagimeli — APPROVED** [review]
Doc and editorial reviews completed

**Pull request merged**